src/bindings/Vscode.mli

```diff
@@ -211,7 +211,7 @@ module Workspace : sig
 
   type cancellationToken = { isCancellationRequested : bool }
 
-  val name : unit -> string option
+  val name : string option
```

Comment:
As far as I understand, `name` is a mutable field. That is, we cannot guarantee that:

```
let x = Workspace.name in
let y = Workspace.name in
assert (x = y)
```

Since ocaml's strings are immutable (with safe string), this means that exposing the binding as `val name : string option` exposes a mutable value as an immutable one. This seemed like a bad idea to me, so I added the `unit` argument so that callers are not confused. What do you think about this?

Reply:
This was resolved in private.